doc/audit/builtin_maps.rst

```diff
@@ -143,6 +143,15 @@ Versions of the code allowed to join the current network.
 
 .. note:: On versions ``3.0.0-dev1`` and below, value was an enum with one possible value ``ALLOWED_TO_JOIN``
 
+``nodes.security_policies``
+~~~~~~~~~~~~~~~~~~
+
+Security policies that nodes can run and join the current network.
+
+**Key** Security Policy Digest: Hash of the security policy string which is reported by the SNP attestation.
```

Comment:
This should state which attestation fields the security policy digest is reported in.

Reply:
This is a bit similar to a discussion I had with Julien about maybe calling it host_data rather than security policy, I think it would make sense to keep the table documentation separate from the specific way in which it's attained in different platforms? As far as I understand, other platforms may have different usages of these fields right? 

I could mention we get it from host_data for the SNPCC case?

Reply:
Yes, that would be good, and yes it can differ on other platforms and we'll need to document that if/when it happens.